Firmware/ultralcd.cpp

```diff
@@ -7417,6 +7417,9 @@ void menu_lcd_lcdupdate_func(void)
 		}
 		else
 		{
+			if(GetPrinterState() == PrinterState::IsSDPrinting) {
+				lcd_print_stop();
```

Comment:
While I'm sure this stop the print fine, I'd be very careful though with doing this inside `menu_lcd_lcdupdate_func`. If `lcd_print_stop` is triggering lcd_update anywhere then you may be creating a recursion to `menu_lcd_lcdupdate_func`.

I don't see any calls to LCD update by looking at the code.